### DISCUSSION TITLE
Potential API solution?

### DISCUSSION THREAD

**Discussion by ItsRogueRen** [Docile-Alligator/Infinity-For-Reddit#1499] [Ideas]
Supposedly this is an update from Reddit stating noncommercial uses can have free API access. Would Infinity count?

https://t.co/vsVhc69vec

**Comment by gitdrug** [comment]
The only way this app or others like it will stay up is if Reddit quits being morons.

**Comment by nelmstr** [comment]
This is exactly what I was thinking.  In order for this to work, each user would have to sign up for personal API access, and Infinity would have to have a way for users to input API codes.

**Reply by alex-devg** [reply]
wouldn't it be safer to migrate though?

Why wouldn't they change their mind and put more limitation on the API, would be exaushting for the developpers to always try to find a way around.

Well i don't know how much work it would mean to change from reddit API to fediverse equivalent, and we'll maybe not the point for the devs
